lib/compiler.js

```diff
@@ -11,7 +11,15 @@
  */
 
 const debug = require('debug')('virtual-jade:compiler')
-const assert = require('assert')
+
+function assertWithDetailedError(node, assertion, message) {
+  if (!assertion) {
+    let err = new Error(message)
```

Comment:
Looks like a few minor linter errors slipped through, like making this `const`: https://travis-ci.org/tdumitrescu/virtual-jade/jobs/145148059